src/Shared/Controls/Settings/MusicSources/ConnectNew/Ipfs/FolderSelector.xaml

```diff
@@ -0,0 +1,100 @@
+﻿<Page
+    x:Class="StrixMusic.Controls.Settings.MusicSources.ConnectNew.Ipfs.FolderSelector"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:behaviors="using:OwlCore.WinUI.Behaviors"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    xmlns:core="using:Microsoft.Xaml.Interactions.Core"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:interactivity="using:Microsoft.Xaml.Interactivity"
+    xmlns:labs="using:CommunityToolkit.Labs.WinUI"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:storage="using:StrixMusic.Controls.Storage"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+    mc:Ignorable="d">
+
+    <Grid
+        MaxWidth="1000"
+        Margin="20"
+        RowSpacing="20">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+
+        <!--<StackPanel Visibility="{x:Bind InvertBoolToVisibility(AuthenticateCommand.ExecutionTask.IsFaulted), Mode=OneWay, FallbackValue=Visible}">
```

Comment:
This looks like leftover code, possibly copied from the OneDrive FolderSelector. Let's just remove this.